_rules/form-control-label-descriptive-cc0f0a.md

```diff
@@ -71,114 +77,130 @@ _There are no major accessibility support issues known for this rule._
 
 #### Passed Example 1
```

Comment:
Most of these examples don't need to be changes. Changing them invalidates a lot of test data. Please keep the examples the way they are, except for changes related to the changes you made in applicability / expectations.